### ISSUE TITLE
3.3.2: provide failure examples/techniques for `placeholder` and `<select>` relying on default `<option>` only

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#2509]
tangential to https://github.com/w3c/wcag/issues/2484 it seems nowhere in the understanding, nor the techniques, do we have anything about what I believe is now the commonly accepted idea that `placeholder` alone fails 3.3.2 as it's not a permanent label/instruction, and along similar lines, having a `<select>` that is only labelled by virtue of its initial default `<option>`.

https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html

suggest that we could do with a few more examples or failure techniques along these lines.

x-ref https://github.com/w3c/wcag/issues/864 and https://github.com/w3c/wcag/issues/1136

**Comment by detlevhfischer** [comment]
Here is a PR #2532 to adress this, and this is a [rendered version of the changed 3.3.2 understanding text](https://raw.githack.com/w3c/wcag/648209001cae1500e06206d1132edf4e109cdb50/understanding/20/labels-or-instructions.html) with changes icluded (2nd and 3rd para in section Intent)

**Comment by patrickhlauke** [comment]
as this topic keeps bubbling up...any chance we could get this looked at at some point?

**Comment by patrickhlauke** [comment]
another good example of a potential failure of 3.3.2 would be form fields that rely solely on the use of `title` as visible label...

**Comment by A11yTea** [comment]
Hi everyone, I am currently writing an issue for this failure and was surprised that this has not been merged yet. I looked at it and it looks good. I would just change "form control" for "input".

**Comment by patrickhlauke** [comment]
> surprised that this has not been merged yet

the AGWG/WCAG 2.x backlog TF moves in mysterious (often surprisingly slow) ways ;)

**Comment by A11yTea** [comment]
Hi @patrickhlauke !
Do you know if this got lost in the mysterious "AGWG/WCAG 2.x backlog TF" void?

**Comment by bruce-usab** [comment]
@A11yTea would you be willing to draft a proposed failure technique?

I just did a quick search to see what I could find as the bones of failure:

From understanding [contrast minimum](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum) and  [contrast enhanced](https://www.w3.org/WAI/WCAG22/Understanding/contrast-enhanced.html):
> This success criterion applies to text in the page, including placeholder text and text that is shown when a pointer is hovering over an object or when an object has keyboard focus. If any of these are used in a page, the text needs to provide sufficient contrast.

From [understanding label in name](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name):
> Note that placeholder text within an input field is not considered an appropriate means of providing a label. The [HTML Standard](https://html.spec.whatwg.org/multipage/input.html#the-placeholder-attribute) states The placeholder attribute should not be used as an alternative to a label. However, it is worth noting that "label" in that statement is in code brackets and links to the label element. For the purposes of this Label in Name Success Criterion, "label" is not used in such a programmatic sense but is simply referring to a text string in close visual proximity to a component. As such, in the absence of any other nearby text string (as described in the preceding list), if an input contains placeholder text, such text may be a candidate for Label in Name. This is supported both through the accessible name calculation (discussed later) and from the practical sense that where a visible label is not otherwise provided, it is likely that a speech-input user may attempt to use the placeholder text value as a means of interacting with the input.

**Comment by patrickhlauke** [comment]
> Hi @patrickhlauke !
> Do you know if this got lost in the mysterious "AGWG/WCAG 2.x backlog TF" void?

less mysterious now we made it public https://github.com/orgs/w3c/projects/56

you'll see we have 450+ items there across todo/in progress/draft ... so things may take a while sometimes. ready-made pull requests stand a better chance than issues that then require somebody to actually make a PR first...

**Comment by A11yTea** [comment]
Sounds good, I think I can try tackling it not this weekend but the next one. You can assign this to me now.

**Comment by bruce-usab** [comment]
Thanks @A11yTea!  No time pressure from our end.  We are all volunteers!
